src/container/srv_container.c

```diff
@@ -2058,8 +2058,10 @@ cont_lookup(struct rdb_tx *tx, const struct cont_svc *svc, const uuid_t uuid, st
 	d_iov_set(&tmp, NULL, 0);
 	/* check if the container exists or not */
 	rc = rdb_tx_lookup(tx, &svc->cs_conts, &key, &tmp);
-	if (rc != 0)
+	if (rc != 0) {
+		DL_ERROR(rc, "cont " DF_UUID ", rdb_tx_lookup failed.\n", DP_UUID(uuid));
```

Comment:
I don't know if adding an ERR message in this internal function is good - maybe it could cause a bunch of unwanted logging in certain cases? There are many callers, and it seems there are differences in the callers, whether they invoke DL_ERROR if this function returns rc != 0.